Сетка страниц и крупных блоков на флексах

as-eliseev opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/33/user/1852073/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/1852073-sedona-33/5/

e1iseev committed:
Сетка страниц и крупных блоков на флексах

as-eliseev committed:
Merge branch 'htmlacademy-htmlcss:master' into master

as-eliseev committed:
Delete index-copy.html

nookismile reviewed (commented):
Блоки не в центровщике
![image](https://user-images.githubusercontent.com/52919340/153225914-696356b5-c4ed-4477-96fa-a0ed4d9f1852.png)

не стилизован телефон
![image](https://user-images.githubusercontent.com/52919340/153226140-75212b67-6c36-422a-a236-ee53273ab352.png)

Центровщик 1056px, не 1200 
![image](https://user-images.githubusercontent.com/52919340/153226736-42afa797-64ac-485c-a55d-d530b25afdc6.png)

e1iseev committed:
Попытка исправления замечаний

1. Блоки разместил по центру
2. Телефон не получается стилизовать через  text-decoration: none;
3. Центровщик поменял на 1056px

e1iseev committed:
Merge branch 'master' of https://github.com/as-eliseev/1852073-sedona-33

e1iseev committed:
исправляет центровку страницы index
